Dockerfile.minimal

```diff
@@ -0,0 +1,17 @@
+FROM nvidia/cuda:9.2-base-ubuntu18.04
+
+ADD ./setup.py /tmp
+ADD ./substratools /tmp/substratools
+ADD ./README.md /tmp
+
+RUN apt-get update; \
+    apt-get install -y python3-minimal python3-pip; \
+    cd /tmp && pip3 install . ; \
+    apt-get remove -y python3-pip --purge --autoremove; \
+    apt-get clean; \
+    rm -rf /var/lib/apt/lists/
+
+RUN mkdir -p /sandbox
+ENV PYTHONPATH /sandbox
+
+WORKDIR /sandbox
```

Comment:
Is the `WORKDIR` instruction really necessary ? According to the documentation it is only used by future `RUN` or `COPY` instructions for example.

Also according to the documentation,`WORKDIR` creates the path if it does not exist, maybe we could replace this line:
```Dockerfile
RUN mkdir -p /sandbox
```
by a simple `WORKDIR`, but it would probably be less explicit.

Dockerfile documentation: https://docs.docker.com/engine/reference/builder/#workdir

Reply:
Thanks! Fixed in https://github.com/SubstraFoundation/substra-tools/pull/41/commits/7ed4815538674b3a7a6fe648d21cb2c5ada2371c

The `WORKDIR` also sets the cwd that will be in effect when you `run` it

```bash
$ docker run -it substrafoundation/substra-tools:0.6.0-minimal bash
root@37015590ea73:/sandbox#
```

So I used your other suggestion to remove the `mkdir`.